src/chainparams.cpp

```diff
@@ -156,6 +156,7 @@ class CMainParams : public CChainParams {
         vSeeds.push_back(CDNSSeedData("dashjr.org", "dnsseed.bitcoin.dashjr.org"));
         vSeeds.push_back(CDNSSeedData("bitcoinstats.com", "seed.bitcoinstats.com"));
         vSeeds.push_back(CDNSSeedData("xf2.org", "bitseed.xf2.org"));
+        vSeeds.push_back(CDNSSeedData("bitcoin.jonasschnelli.ch", "seed.bitcoin.jonasschnelli.ch", true));
```

Comment:
This seed is running in a gnu screen session over a shell `until` `do` script and will startup in case of a server reboot (over `rc.local`).
It's a non-virtual dedicated linux server with recent debian version (Debian 7.8) using a Intel(R) Xeon(R) CPU E31245 @ 3.30GHz with 16GB RAM.
Next to the dns-seed the server is also used for automatic gitian builds as well as for a bitcoind mainnet full-node.